tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "types": ["vite/client", "vite-plugin-pwa/client"],
+    "types": ["vite/client", "vite-plugin-pwa/client","node"],
```

Comment:
was this change intended?

Reply:
Yes

Reply:
Could you elaborate more? state the reason and what it solves? Just curious on what this does.

Reply:
It is used to ensure TypeScript recognizes Node.js globals and core modules, preventing related errors.And it solves issues related to unrecognized Node.js globals (like process), core modules (like fs and path), and environment variables in TypeScript.
Example of the error which i saw are " Cannot find name 'process', __dirname, or require",
Cannot find module 'fs' or its corresponding type declarations etc

Reply:
care_fe is strictly client-side based and doesn't / shouldn't be using any nodejs specific globals or modules directly.

cc: @sainak

Reply:
Thank you for pointing that out! I did use ChatGPT to explore a solution for the issue, which led to including "node" in the types array

Reply:
I tried resolving the issue without including Node.js, but I encountered errors related errors

Reply:
Thats why i asked help for chatgpt

Reply:
what error exactly?

Reply:
The error occurred before I implemented the useDebounce.ts solution, which prompted me to include "node" in the types array as a temporary fix. After implementing the useDebounce logic, I overlooked removing "node" from the configuration. I’ve rechecked the issue now, and it’s working perfectly without needing Node.js. Apologies for the oversight, and thank you for pointing it out!